### PR TITLE
Initialscreen

### PR DESCRIPTION
**Pull request by f-gate**
Entire build.

### CODE REVIEW

File: initialscreen.css [hunk 1]
```diff
@@ -0,0 +1,635 @@
...
+}
+.descrip-holder {
+    position: absolute;
+    width: 100%;
+    height: 50%;
+    margin-top: 10%;
+    margin-bottom: 50%;
+}
+.name {
+    font-size: 3vw;
```
Comment (Benson-J) [inline initialscreen.css]: Using vw can make some text tiny on mobile screens, consider using fixed font size below a breakpoint.

File: initialscreen.css [hunk 2]
```diff
@@ -0,0 +1,635 @@
...
+    color: #CCFBFE;
+    font-weight: initial;
+    border-style: solid;
+    border-width: 0 0 0.1rem 0;
```
Comment (Benson-J) [inline initialscreen.css]: Avoid using rem for anything other than font size

File: initialscreen.css [hunk 3]
```diff
@@ -0,0 +1,635 @@
...
+    .stickynav {
+        height: 10px;
+    }
+    .picturelink {
+        width: 73.3px;
+        height: 46.7px;
+    }
+}
+
+/*! normalize.css v8.0.0 | MIT License | github.com/necolas/normalize.css */
```
Comment (Benson-J) [inline initialscreen.css]: Normalize should go in its own file, & be called before your css.

File: initialscreen.css [hunk 4]
```diff
@@ -0,0 +1,635 @@
...
+    }
+    #byeblock {
+        visibility: hidden;
+    }
+}
+@media only screen and (max-width: 768px) {
+    body {
+        font-size: 64%;
+    }
+    .stickynav {
```
Comment (Benson-J) [inline initialscreen.css]: Doesn't stick on IE10

**Review by Benson-J — COMMENTED** [review]
Good, made a few suggestions.  Remember to stop projects overflowing onto "Contact".

File: initialscreen.css (hunk 2 above)
Comment (f-gate) [reply initialscreen.css]: every rem is gone now thanks.

File: initialscreen.css (hunk 3 above)
Comment (f-gate) [reply initialscreen.css]: placed it before my css

**Pull request closed without merging**